kolibri/content/management/commands/importchannel.py

```diff
@@ -34,7 +36,7 @@ def add_arguments(self, parser):
             help="Download the database for the given channel_id."
         )
 
-        default_studio_url = settings.CENTRAL_CONTENT_DOWNLOAD_BASE_URL
+        default_studio_url = conf.OPTIONS['Urls']['CENTRAL_CONTENT_BASE_URL']
```

Comment:
A few questions and comments

1. In the future if we're allowing people to manage a list of these values from the web UI, would there still be one default value coming from settings? Or would we eventually deprecate this setting? Or... do we move to a place where the ini is actually updated based on modifications to device settings from the web UI?
2. We don't really use the term 'central' [in the UI](http://kolibri.readthedocs.io/en/latest/manage.html#manage-content). We call it 'Studio', though maybe we don't want to do that here? something like "content import server" might be more aligned with existing terminology
3. Any benefit to making this two settings (hostname and protocol), and losing the potentially ambiguous "base url" framing?

Reply:
>In the future if we're allowing people to manage a list of these values from the web UI, would there still be one default value coming from settings? Or would we eventually deprecate this setting? Or... do we move eventually to a setup where the .ini file is actually updated based on modifications to device settings from the web UI?

My thinking: there's always still the single central "hard-coded" (ini-coded) one. The user-provided ones are stored in the database.

>We don't really use the term 'central' in the UI or documentation. We call it 'Studio', though maybe we don't want to do that here? something like "content import server" might be more aligned with existing terminology

Don't see a perfect answer, yeah. It won't always be Studio, could be e.g. the country-level NITA server in Uganda. Don't really want to encourage the mindset of "many Studios", either. But I don't imagine this will be a heavily utilized option for end users anyway.

>Any benefit to making this two settings (hostname and protocol), and losing the potentially ambiguous "base url" framing?

We did this in KA Lite and I've regretted it ever since -- became a big headache to maintain. Beyond those two components, there's also port, path, etc. We're using "base url" phrasing in Morango as well.